site/assets/scss/_navbar.scss

```diff
@@ -4,6 +4,12 @@
   background-image: linear-gradient(to bottom, rgba(var(--bd-violet-rgb), 1), rgba(var(--bd-violet-rgb), .95));
   box-shadow: 0 .5rem 1rem rgba(0, 0, 0, .15), inset 0 -1px 0 rgba(0, 0, 0, .15);
 
+  .bd-navbar-toggle {
+    @include media-breakpoint-down(lg) {
```

Comment:
Bootstrap logo is not centered at some breakpoints. It would be more something like that if we want to have it always centered whatever the width of the window under lg:
Don't trust that code, it just shows the idea:
```css
.bd-navbar-toggle {
  @include media-breakpoint-down(lg) {
    width: 10.25rem;
  }
  @media (max-width: 750px) { // Change of rendering of the search element is not linked to our md
    width: 4.25rem;
  }
}
```